src/diffusers/models/transformers/auraflow_transformer_2d.py

```diff
@@ -451,6 +452,7 @@ def forward(
         hidden_states: torch.FloatTensor,
         encoder_hidden_states: torch.FloatTensor = None,
         timestep: torch.LongTensor = None,
+        joint_attention_kwargs: Optional[Dict[str, Any]] = None,
```

Comment:
Let's rename this to `attention_kwargs`.

Reply:
Done in https://github.com/huggingface/diffusers/pull/10216/commits/6514892fd53f06502329b3927714f7d166db89fd